spring-cloud-dataflow-server-core/src/main/resources/org/springframework/cloud/dataflow/server/db/migration/h2/V1__INITIAL_SETUP.sql

```diff
@@ -88,15 +88,15 @@ CREATE TABLE TASK_LOCK (
 );
 
 CREATE TABLE BATCH_JOB_INSTANCE (
-  JOB_INSTANCE_ID BIGINT IDENTITY NOT NULL PRIMARY KEY,
+  JOB_INSTANCE_ID BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
In general, H2 has gotten stricter about what they allow and the syntax/parsing: this is how you do `IDENTITY` now